diff -ur 0/react-autosuggest/dist/Autosuggest.js 1/react-autosuggest/dist/Autosuggest.js
--- 0/node_modules/react-autosuggest/dist/Autosuggest.js	2018-01-26 19:11:45.398398248 +0000
+++ 1/node_modules/react-autosuggest/dist/Autosuggest.js	2018-01-26 19:11:47.638418011 +0000
@@ -408,6 +408,8 @@

               break;

+            // Tab
+            case 9:
             // Enter
             case 13:
               {
